### PR TITLE
BROKERS: Insert Group Post

### LINKED ISSUE #238: BROKERS: Insert GroupPost
Implement `ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost);`

### PR DESCRIPTION
**Pull request by salimdeveloper**
closes #238

### CODE REVIEW

File: Taarafo.Core/Brokers/Storages/IStorageBroker.GroupPosts.cs [hunk 1]
```diff
@@ -11,5 +11,6 @@ namespace Taarafo.Core.Brokers.Storages
     public partial interface IStorageBroker
     {
         ValueTask<GroupPost> DeleteGroupPostAsync(GroupPost groupPost);
+        ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost);
```
Comment (ElbekDeveloper) [inline Taarafo.Core/Brokers/Storages/IStorageBroker.GroupPosts.cs:14]: Methods should be in order of CRUD operations. So, it should be 
`InsertGroupPostAsync()`
`DeleteGroupPostAsync()`

File: Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs [hunk 2]
```diff
@@ -24,6 +24,18 @@ public async ValueTask<GroupPost> DeleteGroupPostAsync(GroupPost groupPost)
 
             await broker.SaveChangesAsync();
 
+            return groupPostEntityEntry.Entity;
+        }
```
Comment (ElbekDeveloper) [inline Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs:28]: Empty line here

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs (hunk 2 above)
Comment (salimdeveloper) [reply Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs:28]: Should i add an empty line? Please help me understand thanks.

Comment (ElbekDeveloper) [reply Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs:28]: Yes, you did it correctly

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**